src/cloudsc2_nl_loki/cuadjtqs.F90

```diff
@@ -0,0 +1,281 @@
+! Copyright (C) 2003- ECMWF
+!
+! This software is licensed under the terms of the Apache Licence Version 2.0
+! which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+!
+! In applying this licence, ECMWF does not waive the privileges and immunities
+! granted to it by virtue of its status as an intergovernmental organisation
+! nor does it submit to any jurisdiction.
+!
+SUBROUTINE CUADJTQS &
```

Comment:
I believe this file has been manually inlined in the kernel, no? Shouldn't it also come out of the common lib, if this were not the case?